Small UI improvements

aholachek opened the pull request:
Hi, I came across this repo while browsing through Outreachy projects on the Mozilla wiki. I made a few small tweaks to the UI:
- increasing default font sizes for readability
- revamping the landing page styles
- adding a bootstrap variable override less file in /lib so that it is easier to customize other bootstrap styles in the future

Please let me know if there's anything I can do to improve it!

![screen shot 2016-03-10 at 10 57 50 pm](https://cloud.githubusercontent.com/assets/3680488/13693034/ad21c5ba-e714-11e5-81b6-f8819ba0fd28.png)

djmitche commented:
Visually, I love everything about this, especially the landing page, *except* the bigger font size.  As it is, the font size matches that on most other sites (Github, for example) and as a frequent user of this site, I value the screen real-estate and the ability to do things with minimal scrolling.

In terms of implementation -- `custom-bootstrap-variables.less` is really long.  Are all of those customizations, or is this some kind of boilerplate within which you have changed a few things?  Is this the typical approach for customizing bootstrap?  As you can see, I know very little about this myself!  I would prefer to have the customizations only show the things that are actually customized from the bootstrap defaults, but I'm open to learning about the "typical approach" :)

@walac said he would do a more thorough review as well, so assigning to him.

walac commented on landingpage.jsx:
```diff
@@ -37,16 +37,18 @@ $(function() {
+          <bs.Col md={8} mdOffset={2} sm={10} smOffset={1}>
+            <div className="landingpage-header">
+              <img src={"/lib/assets/taskcluster-180.png"}/>
+              <h2><span className="light-font">Welcome to</span> <span className="landingpage-logo">TaskCluster Tools</span></h2>
```
nit: anyway to fit this in 80 characters column size?

walac commented on landingpage.jsx:
```diff
@@ -37,16 +37,18 @@ $(function() {
+        <bs.Row className="landingpage-description">
+          <bs.Col sm={12}>
+            <p>
+              A collection of tools for TaskCluster components and elements in the TaskCluster eco-system. Here you'll find tools to manage TaskCluster as well as run, debug, inspect and view tasks, task-graphs, and other TaskCluster related entities.
```
nit: anyway to fit this in 80 characters column size?

aholachek commented:
Hi, I was on my lunch break so decided to make some changes in response to your suggestions.
- On personal projects I guess it's nice to have copied over the entire bootstrap variables file so that you can play around with it easily, but I see how on larger projects that's a potentially very messy approach, so I updated the vars file to only contain what has been changed.
- I also changed the base font size down to 15px, by using 16px I was following the [advice found here from Google] (https://developers.google.com/speed/docs/insights/UseLegibleFontSizes)

walac commented:
Just a couple of code formatting nits, but this looks fantastic! Thank you very much \o/
For follow up on outreachy program, reach me (wcosta) or @djmitche (dustin) out in #outreach or #taskcluster in irc.mozilla.org

djmitche commented:
Looks like our complaints are fixed.  I'm just loading it up one more time (grunt, y u so slooow!!)..

djmitche commented:
This is awesome - thanks!

djmitche commented:
That should be live on the site in 20 minutes or so.
